pkgs/stdenv/darwin/make-bootstrap-tools.nix

```
@@ -79,6 +87,7 @@ in rec {
 
       cp -rL ${llvmPackages.clang-unwrapped}/lib/clang $out/lib
 
+      cp -d ${llvmPackages.libclang}/lib/libclang-cpp*.dylib $out/lib
       cp -d ${llvmPackages.libcxx}/lib/libc++*.dylib $out/lib
       cp -d ${llvmPackages.libcxxabi}/lib/libc++abi*.dylib $out/lib
       cp -d ${llvmPackages.llvm.lib}/lib/libLLVM.dylib $out/lib
```

Since 7869d1654517c028aa76fc1dedc9b5ac301a867a something like this would be required:

```suggestion
      cp -rL ${llvmPackages.libclang.lib}/lib/clang $out/lib

      cp -d ${llvmPackages.libclang.lib}/lib/libclang-cpp*.dylib $out/lib
      cp -d ${llvmPackages.libcxx.lib}/lib/libc++*.dylib $out/lib
      cp -d ${llvmPackages.libcxxabi.lib}/lib/libc++abi*.dylib $out/lib
      cp -d ${llvmPackages.llvm.lib}/lib/libLLVM.dylib $out/lib
```

---

`lib.getLib` is more future proof: `${lib.getLib llvmPackages.libcxxabi}`

---

I've applied the suggestion everywhere a `/lib` is used. Thanks!